core/src/apps/cardano/layout.py

```diff
@@ -440,13 +433,10 @@ async def show_warning_tx_network_unverifiable(ctx: wire.Context) -> None:
 async def show_warning_address_pointer(
     ctx: wire.Context, pointer: CardanoBlockchainPointerType
 ) -> None:
-    await show_warning(
-        ctx,
-        (
-            "Pointer address:",
-            "Block: %s" % pointer.block_index,
-            "Transaction: %s" % pointer.tx_index,
-            "Certificate: %s" % pointer.certificate_index,
-        ),
-        button="Ok",
-    )
+    text = Text("Warning", ui.ICON_WRONG, ui.RED)
```

Comment:
keeping the icon for consistency with previous UI but maybe there's a more suitable icon as technically there's nothing "wrong" per se